tests/unit/QuantFile.spec.js

```diff
@@ -289,4 +288,49 @@ describe('QuantFile.vue', () => {
     })
   })
 
+  describe('tapestation X10 VDJ file', () => {
+    beforeEach(async () => {
+      quantFile = new cmp({propsData: { quant: 'library10XVDJ', filename: 'DN987654 - 2019-11-05 - 16-32-56-D5000_compactRegionTable(1).csv'}})
+      plate = fs.readFileSync('./tests/data/DN987654 - 2019-11-05 - 16-32-56-D5000_compactRegionTable(1).csv', 'ascii')
+      file = new File([plate], 'DN987654 - 2019-11-05 - 16-32-56-D5000_compactRegionTable(1).csv', { type: 'text/plain'})
+    })
+
+    it('will have a filename', () => {
+      expect(quantFile.filename).toEqual('DN987654 - 2019-11-05 - 16-32-56-D5000_compactRegionTable(1).csv')
```

Comment:
Should the filename have this many spaces in it!?

Reply:
The example file I was given was exactly this name.  The detection code for the plate barcode doesn't mind either way.

Reply:
That '(1)' on the end of the filename is just a Mac thing when you download the same file twice, I would ditch that at least.
Filenames on these instruments are generally set up in config screens as combinations of variables (e.g. looks like <barcode> - <date> - <time>_compactRegionTable.csv). Possibly worth getting them to remove the spaces they've added in there. But if it's not causing a problem no big deal.